docs/getting-started-guides/kubeadm.md

```diff
@@ -218,14 +218,16 @@ See the [list of add-ons](/docs/admin/addons/) to explore other add-ons, includi
 
 * To uninstall the socks shop, run `kubectl delete -f microservices-demo/deploy/kubernetes/manifests` on the master.
 
-* To undo what `kubeadm` did, simply delete the machines you created for this tutorial, or run the script below and then uninstall the packages.
-  <details>
-     <pre><code>systemctl stop kubelet;
-  docker rm -f $(docker ps -q); mount | grep "/var/lib/kubelet/*" | awk '{print $3}' | xargs umount 1>/dev/null 2>/dev/null;
-  rm -rf /var/lib/kubelet /etc/kubernetes /var/lib/etcd /etc/cni;
-  ip link set cbr0 down; ip link del cbr0;
-  ip link set cni0 down; ip link del cni0;
-  systemctl start kubelet</code></pre>
+* To undo what `kubeadm` did, simply delete the machines you created for this tutorial, or run the script below and then start over or uninstall the packages.
+
+  <br><br><details>
```

Comment:
I think the details tag should be removed, the code below clearly needs more visibility since many users haven't found it.

Anyway, now it doesn't look good: http://deploy-preview-1426.kubernetes-io-vnext-staging.netlify.com/docs/getting-started-guides/kubeadm/